config/locales/pt.yml

```diff
@@ -97,9 +97,12 @@ pt:
   disclaimer:
     title: "Organização"
 
-  misc:
+  genders:
     male: 'masculino'
     female: 'feminino'
+    other: 'inny'
```

Comment:
@cubizh, does this sound correct in Portuguese?

Reply:
No, please change it to 'outro'